packages/core/test/components/DataTable.test.tsx

```diff
@@ -118,8 +118,8 @@ const data: ParentRow[] = [
   {
     data: {
       name: 'Frontend Fabien',
-      jobTitle: 'Engineer',
-      tenureDays: 600,
+      jobTitle: undefined,
+      tenureDays: null,
```

Comment:
add a test that sorting returns `null` and `undefined` at the end? add a sort test?